configure.ac

```diff
@@ -548,12 +546,41 @@ if test -z "$ENABLE_SOS_TRUE"; then
 	fi
 fi
 
-dnl ldms-python API's
+dnl ldms-python API's and programs
+AX_WITH_PROG([CYTHON], [cython], [no])
+CYTHON_REQ_VER="0.28.5"
+CYTHON_MSG="
+
+    Please install Cython ver >= ${CYTHON_REQ_VER}. It is recommended to install
+    Cython via pip3 (\`pip3 install Cython\`) to get the latest version of
+    Cython.  Some installation package (e.g. python36-Cython from epel) may
+    install cython program with cython<VER> form (e.g. /bin/cython3.6). In such
+    case, --with-cython=FULL_PATH_TO_CYTHON_PROGRAM option is required (e.g.
+    --with-cython=/bin/cython3.6).
+
+"
 if test -z "$ENABLE_LDMS_PYTHON_TRUE"; then
 	AX_PYTHON_DEVEL([>='3.6'])
 	pkgpythondir="${pythondir}/ovis_ldms"
 	pkgpyexecdir="${pkgpythondir}"
 	AX_PYTHON_MODULE_VERSION(["argparse"], 1.1, "python")
+
+	dnl cython check
+	if test "${CYTHON}" == "no"; then
+		dnl cython not found, try cython3 or cython3.6
+		AC_MSG_NOTICE(cython not found ... trying cython3 or cython3.6)
+		CYTHON=$(which cython3 2>/dev/null || \
+			 which cython3.6 2>/dev/null || \
```

Comment:
EPEL package for RHEL4 is cython-3.8 (Yes, that one really has a dash in the name, sigh). We'll need that too.

Reply:
Do we support RHEL4?

Reply:
Sorry, brain fart. I meant RHEL8. (TOSS4 is RHEL8-based, which is where the "4" snuck in.)